### DISCUSSION TITLE
Is there a way to enable spreading lava without a lava bucket?

### DISCUSSION THREAD

**Discussion by goardis** [Pryaxis/TShock#1797] [Q&A]
Using the new lava items triggers some cheat detection and removes the lava and freezes you. Is there a way to disable this until it's fixed?

**Comment by hakusaro** [comment]
Which items?

**Reply by goardis** [reply]
Bottomless Lava Bucket and Lava Bombs

**Comment by moisterrific** [comment]
Yeah those new items need to be accounted for in bouncer and IIRC there are also lava rockets!

**Comment by moisterrific** [comment]
Bottomless Lava Bucket 
Item ID: 4820
https://terraria.gamepedia.com/Bottomless_Lava_Bucket

Lava Bomb
Item ID: 4825
Projectile ID: 904
https://terraria.gamepedia.com/Lava_Bomb

Couldn’t find lava rockets on the wiki but I swear I seen them in builder’s workshop…or maybe I mistook lava bombs for them  ¯\_(ツ)_/¯
